runtime/dom-slot.js

```diff
@@ -46,13 +46,13 @@ class DomSlot extends Slot {
     return infos;
   }
   // TODO(sjmiles): SlotManager calls here
-  render(content, eventHandler) {
+  render(content, arc, eventHandler) {
```

Comment:
Plumbing the arc through feels like too-tight coupling between the objects. I suspect we can avoid this as described above.